single-line-diagram/single-line-diagram-core/src/main/java/com/powsybl/sld/svg/styles/AbstractVoltageStyleProvider.java

```diff
@@ -47,12 +45,11 @@ protected AbstractVoltageStyleProvider(BaseVoltagesConfig baseVoltagesConfig) {
 
     @Override
     public List<String> getEdgeStyles(Graph graph, Edge edge) {
-        List<String> list = Stream.of(getVoltageLevelEdgeStyle(graph, edge), getDanglingLineStyle(edge))
-                .filter(Optional::isPresent)
-                .map(Optional::get)
-                .collect(Collectors.toList());
-        list.add(0, WIRE_STYLE_CLASS);
-        return list;
+        List<String> edgesStyles = new ArrayList<>();
+        edgesStyles.add(WIRE_STYLE_CLASS);
+        edgesStyles.addAll(getVoltageLevelEdgeStyle(graph, edge));
+        getDanglingLineStyle(edge).ifPresent(edgesStyles::add);
+        return List.copyOf(edgesStyles);
```

Comment:
You can return directly the list created here, there's no risk
```suggestion
        return edgesStyles;
```